Document delete deployment feature (#2219)
mintlify/docs: deploy/deployments.mdx

  </Frame>
  </Step>
</Steps>

## Deleting a deployment

> You can permanently delete a deployment from the Danger Zone in your dashboard settings. This action is irreversible and will remove all deployment data, including any associated preview deployments.

Reviewer: ```suggestion
You can permanently delete a deployment from the [Danger zone](https://dashboard.mintlify.com/settings/organization/danger-zone) in your dashboard settings. This action is irreversible and will remove all deployment data, including any associated preview deployments.
```

<Steps>
  <Step title="Navigate to the Danger Zone.">

>     Go to your [dashboard](https://dashboard.mintlify.com), then navigate to **Settings** > **Organization** > **Danger zone**.

Reviewer: ```suggestion
    Go to the [Danger zone](https://dashboard.mintlify.com/settings/organization/danger-zone) in the settings page of your dashboard.
```
